gcp_variant_transforms/transforms/write_variants_to_shards.py

```diff
@@ -37,7 +37,8 @@ def __init__(self, vcf_shards_output_dir, number_of_variants_per_shard):
     """
     self._vcf_shards_output_dir = vcf_shards_output_dir
     self._number_of_variants_per_shard = number_of_variants_per_shard
-    self._coder = vcfio._ToVcfRecordCoder()
+    # Write shards as is.
+    self._coder = vcfio._ToVcfRecordCoder(use_1_based_coordinate=False)
```

Comment:
Is this for annotation?
Let me make sure I understand this: 

In #635 we modify `_shard_variants()` so that it calls `_read_variants()` with `use_1_based_coordinate=False` which means we read them in 0-based coordinate. Here we again set `use_1_based_coordinate=False` which means when writing to shards we convert back to 1-based (which is VCF expected coordinate) and thus VEP receives variants correctly. This means we could set both to `True` and still annotation would work exactly the same. So basically for annotation we read variants by ignoring the `--use_1_based_coordinate` and we send those variant to VEP for annotation in 0-based coordinate.

If this correct could you make the comment a bit more informative and explain the connection of this False value to the one in `_shard_variants()`.

Reply:
Done.